pkg/subctl/cmd/deploybroker.go

```diff
@@ -73,6 +78,8 @@ func init() {
 
 	deployBroker.PersistentFlags().BoolVar(&operatorDebug, "operator-debug", false, "enable operator debugging (verbose logging)")
 
+	deployBroker.PersistentFlags().StringVar(&brokerNamespace, "broker-namespace", defaultBrokerNamespace, "namespace for broker")
```

Comment:
I know this was merged a while ago, but isn't it simpler UX and more consistent to just name this "--namespace"?

I'm talking about the flag name not the variable name.

Reply:
The name was chosen to make it clear that it sets the broker namespace, as opposed to the operator namespace, or more generally to avoid suggesting the command's effect would only apply to a single `--namespace`

Reply:
Yes that might make sense, but considering we're talking about `subctl deploy-broker` command I think having `broker` in the flag as well is repetitive.

Also, aren't we essentially affecting a single namespace here? So I'm not sure it's incorrect in that regard.